packages/openssl/buildinfo.json

```diff
@@ -1,7 +1,7 @@
 {
   "single_source" : {
     "kind": "url_extract",
-    "url": "https://www.openssl.org/source/openssl-1.1.1c.tar.gz",
-    "sha1": "71b830a077276cbeccc994369538617a21bee808"
+    "url": "https://www.openssl.org/source/openssl-1.1.1d.tar.gz",
+    "sha1": "056057782325134b76d1931c48f2c7e6595d7ef4"
```

Comment:
what about windows? especially in mesos we are depending on it as well!

Reply:
For master, Windows is already using 1.1.1d.
For 2.0, there is no Windows support.